dryCypher.js

```diff
@@ -0,0 +1,54 @@
+// returns a string to be used by the encode function
+// by repeating the encode word as many times as needed
+// to match the length of the message being encoded
+function setupKey(keyword,message){
+  var encodeKey = keyword;
+  while(encodeKey.length < message.length){
+    encodeKey += encodeKey;
+  }
+  while (encodeKey.length > message.length){
+    encodeKey = encodeKey.slice(0,-1);
+  }
+  return encodeKey;
+}
+
+// uses the first letter of the encodekey to encode OR decode
+// the first letter of the message string depending on the value of 'x'
+function alterCharacter(message, encodeKey, x) {
+  if ( x === 0 ) {
```

Comment:
Why not use a boolean variable instead of a number for x? And call it a more meaningful name (see below).